services/file_system.js

```diff
@@ -1,8 +1,20 @@
 const fs = require('fs');
+const url = require('url');
 
 const { rpc, rpc_process } = require('carlo/rpc');
 const chokidar = require('chokidar');
 
+let pathToUrl;
+if (!url.pathToFileURL) {
+  // Node 8 does not convert file paths to file urls.
+  pathToUrl = str => str;
```

Comment:
Shouldn't this be a proper polyfill, that adds "file://" protocol, escape characters, etc?

Reply:
Node 8 does not convert paths reported using protocol. It is required to report it as is here to force DevTools use raw path to set breakpoint using "scriptUrl" that will match actual script in Node..